src/mixins/treeview.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { mapActions, mapState } from 'vuex'
+/* eslint-disable no-unused-vars */
+import User from '@/model/User.model'
+/* eslint-enable no-unused-vars */
+
+const treeview = {
```

Comment:
The name `treeview` is going to cause confusion with the "Tree View". Perhaps `datatree` or something like that?

Reply:
Good point! Changing it tomorrow :+1: